pkg/manifestclient/write_roundtripper.go

```diff
@@ -17,10 +18,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/server"
-	"net/http"
 	"sigs.k8s.io/yaml"
 )
 
+const applicationJsonContentType = "application/json"
```

Comment:
Nit: There's a const in apimachinery for this one (https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime#pkg-constants).

Reply:
Thank you!